ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -499,6 +499,16 @@ function admin_ack() {
 
 # Upgrade the cluster to target release
 function upgrade() {
+    local conditional_updates
```

Comment:
we also need add set_channal step here since `cucushift-upgrade-setchannel` step could not be call in the chain loop